src/componenets/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { Sling as Hamburger } from 'hamburger-react';
+import { AiOutlineBell } from 'react-icons/ai';
+import { RiArrowDownSLine } from 'react-icons/ri';
+import { BiSearch } from 'react-icons/bi';
+import profilePhoto from '../assets/profile-photo.jpeg';
+
+const Navbar = () => {
+  const [isOpen, setOpen] = useState(false);
+
+  return (
+    <div className="navbar">
+      <div className="hamburger">
+        <Hamburger toggled={isOpen} toggle={setOpen} size={20} rounded color="#7e7d7f" />
+        <nav className={isOpen ? 'navlinks Open' : 'navlinks'}>
+          <div className="mob-profile">
+            <img src={profilePhoto} alt="profile" className="profile-photo mob-profile-photo" />
+            <p className="username mob-username">Username</p>
+            <p>example@email.com</p>
+          </div>
+          <NavLink to="/" activeclassname="active" onClick={() => setOpen(false)}>Houses</NavLink>
+          <NavLink to="/MyFavourites" activeclassname="active" onClick={() => setOpen(false)}>My Favorites</NavLink>
+          <NavLink to="/AddHouse" activeclassname="active" onClick={() => setOpen(false)}>Add house</NavLink>
+          <NavLink to="/DeleteHouse" activeclassname="active" onClick={() => setOpen(false)}>Delete house</NavLink>
+          <hr className="divider" />
+          <NavLink to="/LogOut" activeclassname="active" onClick={() => setOpen(false)}>Log Out</NavLink>
```

Comment:
Accessibility: Ensure that your NavLink components have proper aria-label attributes or visible text that describes their purpose, to make the navigation more accessible to users who rely on assistive technologies like screen readers.